test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -76,10 +77,14 @@ private void assertCommandBehavior(String inputCommand,
                                       AddressBook expectedAddressBook,
                                       boolean isRelevantPersonsExpected,
                                       List<? extends ReadOnlyPerson> lastShownList) throws Exception {
+        Command command = new Parser().parseCommand(inputCommand);
 
         //Execute the command
         CommandResult r = logic.execute(inputCommand);
 
+        if(!command.isMorph()){
```

Comment:
Since the behavior of the code has been changed, do ensure that the header comment is updated as well.